src/panels/lovelace/hui-editor.ts

```diff
@@ -125,7 +147,8 @@ class LovelaceFullConfigEditor extends hassLocalizeLitMixin(LitElement) {
       return;
     }
 
-    this.lovelace!.saveConfig(value);
+    await this.lovelace!.saveConfig(value);
```

Comment:
this needs to be in try…catch.

Also if invalid YAML or if saving fails, saving is never reset to false.